javascript/javascript1/week3/Name_Removal/name.js

```diff
@@ -1,8 +1,8 @@
 const names = ['Peter', 'Ahmad', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala'];
 const nameToRemove = 'Ahmad';
 
-// Write some code here
-names.splice(1,1);
-console.log(names);
-
-// Code done
+const indexOfNameToRemove = names.indexOf(nameToRemove) ;
+if (names[indexOfNameToRemove] === nameToRemove)  {
```

Comment:
What if the index was not found? This will crash when it tries to get names[-1]
indexOf already did the search for you, so all you have to do is `if (indexOfNameToRemove >=0)`